.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
 - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
 - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
 - sudo apt-get update && sudo apt-get install yarn
+- docker build dockerfiles/gomoderator
```

Comment:
I think we probably need to run this in addition to building it if we want it to throw an error when the `gomoderator` detects conflicts

Reply:
Am I interpreting [this](https://travis-ci.com/makerdao/vdb-mcd-transformers/builds/150565668) wrong? 
On line 1688 it says `Step 20/20 : RUN python3 scripts/gomoderator.py ./ vdb/` 
followed up by line 1867 `no conflicts to resolve`. 

Is this just running in some intermediate container?

Reply:
Nope, you're right 👍 

I didn't realize we were calling it with `RUN` in the build step